library/src/main/java/com/datatorrent/lib/io/fs/AbstractFileInputOperator.java

```diff
@@ -137,6 +165,7 @@
   private WindowDataManager windowDataManager = new WindowDataManager.NoopWindowDataManager();
   protected transient long currentWindowId;
   protected final transient LinkedList<RecoveryEntry> currentWindowRecoveryState = Lists.newLinkedList();
```

Comment:
So do both start batch and end batch tuples have a end window delivery mechanism?